src/appshell/view/dockwindow/dockpanelview.h

```diff
@@ -54,6 +54,8 @@ class DockPanelView : public DockBase
 
     bool isTabAllowed(const DockPanelView* tab) const;
     void addPanelAsTab(DockPanelView* tab);
+    int getTabIndexOfPanel(DockPanelView* tab);
+    int getCurrentTabIndex();
```

Comment:
Minor code style nitpick: usually we don't use the `get` prefix in method names. So this would become simply `tabIndexOfPanel` and `currentTabIndex`.

Reply:
Thanks, updated.